index.html

```diff
@@ -150,6 +165,69 @@ <h3>Project name goes here</h3>
           </div>
         </div>
       </section>
+
+      <section  id='project' class="desktop-version">
+        <div class="projectgrid">
+          <div class="grid1">
+            <h1 class="title">Projects</h1>
+            <img src="./assets/img/image_dots.png" alt="project1" />
+          </div>
+          <div class="project grid2">
+            <div class="projectimg">
+              <img src="./assets/img/desktop.png" alt="project1" />
+            </div>
+            <div class="projectdetails">
+              <h3>Project name goes here</h3>
+              <ul class="projectbuttons">
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>JavaScript</li>
+              </ul>
+              <div class="viewbutton">
+                <p>See this project</p>
+                <i class="fa-solid fa-arrow-right"></i>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="mixedgrid">
+          <div class="firstimage">
+          <img src="./assets/img/project1.png" id="laptopimage" alt="project6" />
+          </div>
+          <div class="projectimg">
+            <img  id='shapeimages' src="./assets/img/projectimage.png" alt="dots" />
+          </div>
+          <div class="project"  id="detailsproject">
+            <div class="projectdetails" id="projectdetails" >
+              <h4>Project name goes here</h4>
+              <ul class="projectbuttons">
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>+1</li>
+              </ul>
+              <div class="viewbutton">
+                <p>See this project</p>
+                <i class="fa-solid fa-arrow-right"></i>
+              </div>
+            </div>
+          </div>
+          <div id="second-img">
+            <img id='projectimg1' src="./assets/img/laptop.png" alt="project1" />
+          </div>
+          <div id="third-img">
+            <img id='projectimg2' src="./assets/img/shapes2.png" alt="project1" />
+          </div>
+          <div id="fourth-img">
+            <img id='projectimg3' src="./assets/img/icon2.png" alt="project1" />
+            <img id='projectimg4' src="./assets/img/icon3.png" alt="project1" />
+          </div>
+          <div id="fifth-img">
+            <img id='projectimg4' src="./assets/img/icon4.png" alt="project1" />
+          </div>
+          
+        </div>
+        
+      </section>
       <section id="about">
         <div>
           <h1 class="title">About</h1>
```

Comment:
- [ ] Kindly make sure your about section is visible on the Desktop version
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/206683648-b900a906-8e35-41fb-8550-70c4d4bad86e.png)

Figma template:point_down: 
![image](https://user-images.githubusercontent.com/97100091/206683881-4b8f2de6-8dbd-4a38-b6e5-335f023c9f0b.png)